tracer/src/Datadog.Tracer.Native/iast/dataflow.cpp

```diff
@@ -83,6 +83,7 @@ static const WSTRING _fixedAssemblyExcludeFilters[] = {
     WStr("DelegateDecompiler*"),
     WStr("FluentValidation*"),
     WStr("NHibernate*"),
+    WStr("Npgsql*"),
```

Comment:
Interesting 🤔 Should the other libraries be in here too? e.g. MySql etc?

Reply:
MySql is already there (the list is long and not sorted...)